ci-operator/config/openshift/network-tools/openshift-network-tools-master.yaml

```diff
@@ -0,0 +1,46 @@
+base_images:
+  base:
+    name: "4.7"
+    namespace: ocp
+    tag: base
+  ocp_builder_rhel-8-golang-1.15-openshift-4.7:
+    name: builder
+    namespace: ocp
+    tag: rhel-8-golang-1.15-openshift-4.7
```

Comment:
though this new image is being added only in 4.7, do I need to add the previous base image portions for 4.6 ? - that was there for the debug-network image here ? (https://github.com/openshift/release/commit/2cfa61c50ff32d1e7579179095b2dd75b45c0a7f#diff-84750c89aee08542da3edfef1ab43468b1462fa067dcaf2759ac01738fa49c87R1-R9). Asking what will happen to the mirroring in 4.6 once we delete the debug-network image.

Reply:
4.6 will have its own config file in the same folder, named  with the branch name as suffix instead of `master`